AssetMappr/presentation/landingPage.py

```diff
@@ -11,19 +11,20 @@
 from dash import html
 import dash_bootstrap_components as dbc
 
+
 def makeLandingPage():
```

Comment:
Based on how it looks in the pull request review app, the content is not exactly centered, so the position looks a bit funny. Maybe we can also move it a  bit higher so there isn't so much white space on the top? These are very minor points, will wait to see what the rest of the team says as well